app/scripts/metamask-controller.js

```diff
@@ -3556,26 +3682,49 @@ export default class MetamaskController extends EventEmitter {
    * Adds a new account to the default (first) HD seed phrase Keyring.
    *
    * @param accountCount
-   * @returns {Promise<string>} The address of the newly-created account.
+   * @param accountName
+   * @returns {} keyState
    */
-  async addNewAccount(accountCount) {
+  async addNewAccount(accountCount, accountName) {
     const isActionMetricsQueueE2ETest =
       this.appStateController.store.getState()[ACTION_QUEUE_METRICS_E2E_TEST];
 
     if (process.env.IN_TEST && isActionMetricsQueueE2ETest) {
       await new Promise((resolve) => setTimeout(resolve, 5_000));
     }
 
-    const oldAccounts = await this.coreKeyringController.getAccounts();
-
-    const { addedAccountAddress } =
-      await this.coreKeyringController.addNewAccount(accountCount);
-
-    if (!oldAccounts.includes(addedAccountAddress)) {
-      this.preferencesController.setSelectedAddress(addedAccountAddress);
+    const [primaryKeyring] = this.coreKeyringController.getKeyringsByType(
```

Comment:
Check conflix resolution

Reply:
Changed in 88e5656dbfda38689bc179ce5dd40cecd87b39a8